src/draw/handler/Draw.Polygon.js

```diff
@@ -26,7 +26,7 @@ L.Draw.Polygon = L.Draw.Polyline.extend({
 		// Whether to use the metric measurement system (truthy) or not (falsy).
 		// Also defines the units to use for the metric system as an array of
 		// strings (e.g. `['ha', 'm']`).
-		metric: false,
+		metric: true,
```

Comment:
Dit is de default, ook upstream. Ik had het eerst gewijzigd zonder daar erg in te hebben.